locales/en/discord.json

```diff
@@ -6,6 +6,7 @@
 	"BOT_TIMED_OUT": "I am currently timed out.",
 	"BOT_UNSUPPORTED_CHANNEL": "I cannot start a session in this channel type.",
 	"CMD_ERROR": "There was an error while handling the command.",
+	"BUTTON_ERROR": "There was an error while handling the button.",
```

Comment:
"There was an error while processing your request." this error message is directly shown when a user has directly interacted with the button. we could simplify both CMD_ERROR and BUTTON_ERROR to GENERIC_ERROR instead, honestly don't see the need for both